### DISCUSSION TITLE
What is generally considered the "maximum break" in snooker?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#444] [Galaxy Trivia] [answered]
**Bonus:** What is the absolute maximum number of points that can be achieved in a very, very rare case?

**Comment by peterbaccaro** [comment, marked as answer]
A **147** (one-four-seven) is generally considered to be the maximum break in snooker.
A **155** break is the absolute maximum break you can get in snooker and it is very rare. It can only come about when a player fouls at the beginning of the game (when no balls have been potted yet) which leads to a free ball.
